src/js/components/Spinner/Spinner.js

```diff
@@ -98,6 +126,12 @@ const Spinner = forwardRef(
           typeof borderThemeProp === 'undefined' ? defaultBorder : spinnerBorder
         }
         {...themeProps}
+        // If border plus pad is larger than spinnerSize there should be no pad.
+        pad={
+          spinnerPadBorder * 2 > spinnerSize.replace(/px/, '') * 1
```

Comment:
```suggestion
          spinnerPadBorder * 2 > spinnerSize.replace('px', '')
```

Reply:
Should we just include the `*2` above where we calculate `spinnerPadBorder`? and add a comment about why we multiple by 2.

Reply:
yes I can also add